packages/allure-playwright/src/helpers.ts

```diff
@@ -0,0 +1,105 @@
+import { TestInfo } from "@playwright/test";
+import { Label, LabelName, LinkType } from "allure-js-commons";
+
+export const ContentTypes = {
+  label: "allure/label",
```

Comment:
text prefix allows readers interpret it as text

```suggestion
  label: "text/allure-label",
```

Reply:
> text prefix allows readers interpret it as text

is that required? I rather use correct custom content type like `application/vnd.allure.metadata+json`

and use one attachment format for all the test info attachments

```ts
interface Metadata {
  id?: string;
  description?: string;
  links?: Link[];
  labels?: Label[];
}
```

Reply:
That works as well. As long as the content type is sensible, all is good.